rdmo/management/management/commands/merge_attributes.py

```diff
@@ -26,16 +26,24 @@ def add_arguments(self, parser):
             help='The URI of the target attribute that will be used to replace the source'
         )
         parser.add_argument(
-            '--save',
+            '--delete',
```

Comment:
Why both with the same argument, I can see a case where I want to move everthing from one attribute to another one, but keep the old. Does that make sense?

Reply:
ok, the args are now:
```
  --source SOURCE       The URI of the source attribute that will be replaced by the target and will be deleted
  --target TARGET       The URI of the target attribute that will be used to replace the source
  --save                If specified, the changes will be saved. If not specified, the command will not make any changes in the database.
  --delete              If specified, the source attribute will be deleted. If not specified, the command will not delete the source attribute.
  --view                If specified, the changes will be applied to the affected Views as well.
```